### PR TITLE
fix: typo in manage-resources-containers.md

### PR DESCRIPTION
**Pull request by iltyty**

### CODE REVIEW

File: content/en/docs/concepts/configuration/manage-resources-containers.md [hunk 1]
```diff
@@ -114,7 +114,7 @@ resource requests/limits of that type for each container in the Pod.
 {{< feature-state feature_gate_name="PodLevelResources" >}}
 
 Starting in Kubernetes 1.32, you can also specify resource requests and limits at
-the Pod level. the Pod level. At Pod level, Kubernetes {{< skew currentVersion >}}
+the Pod level. At Pod level, Kubernetes {{< skew currentVersion >}}
```
Comment (windsonsea) [inline content/en/docs/concepts/configuration/manage-resources-containers.md:117]:
```suggestion
the Pod level. At the Pod level, Kubernetes {{< skew currentVersion >}}
```
Might be better for consistency.

Comment (iltyty) [reply content/en/docs/concepts/configuration/manage-resources-containers.md]: done

**Review by windsonsea — COMMENTED** [review]
Thanks.
/lgtm

**Review by dipesh-rawat — COMMENTED** [review]
/approve

**Pull request merged**